packages/x-date-pickers/src/MultiSectionDigitalClock/MultiSectionDigitalClockSection.tsx

```diff
@@ -187,13 +188,44 @@ export const MultiSectionDigitalClockSection = React.forwardRef(
 
     const focusedOptionIndex = items.findIndex((item) => item.isFocused(item.value));
 
+    const handleKeyDown = (event: React.KeyboardEvent) => {
+      switch (event.key) {
+        case 'PageUp': {
+          if (!containerRef.current) {
+            return;
+          }
+          const newIndex = getFocusedListItemIndex(containerRef.current) - 5;
+          const children = containerRef.current?.children;
+          const newFocusedIndex = Math.max(0, newIndex);
+
+          (children[newFocusedIndex] as HTMLElement).focus();
```

Comment:
We are playing with HTML here. 🤔 
WDYT about keeping the null safety for those cases where the DOM might have been tampered with? 🤔 
I.e.:
```tsx
const childToFocus = children[newFocusedIndex];
if (childToFocus) {
  childToFocus.focus();
}
```